mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -94,3 +97,39 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 		METHOD <init> (Lnet/minecraft/class_2784;D)V
 			ARG 1 size
 		METHOD method_11996 recalculateBounds ()V
+	CLASS class_5200 Config
+		FIELD field_24123 centerX D
+		FIELD field_24124 centerZ D
+		FIELD field_24125 damagePerBlock D
+		FIELD field_24126 safeZone D
+		FIELD field_24127 warningBlocks I
+		FIELD field_24128 warningTime I
+		FIELD field_24129 size D
+		FIELD field_24130 sizeLerpTime J
+		FIELD field_24131 sizeLerpTarget D
+		METHOD <init> (DDDDIIDJD)V
+			ARG 1 x
+			ARG 3 z
+			ARG 5 damage
+			ARG 7 safeZone
+			ARG 9 warningBlocks
+			ARG 10 warningTime
+			ARG 11 size
+			ARG 13 sizeLerpTime
+			ARG 15 sizeLerpTarget
```

Comment:
```suggestion
			ARG 13 lerpTime
			ARG 15 lerpTarget
```

Reply:
Same as above. I believe `sizeLerpTime/Target` is more correct.